static/app/views/userFeedback/userFeedbackEmpty.tsx

```diff
@@ -31,6 +34,20 @@ export function UserFeedbackEmpty({projectIds}: Props) {
   const hasNewOnboarding = organization.features.includes('user-feedback-onboarding');
   const {activateSidebarIssueDetails} = useFeedbackOnboardingSidebarPanel();
 
+  const router = useRouter();
+  const setProjId = useCallback(() => {
+    router.push({
+      pathname: location.pathname,
+      query: {...location.query, project: projectIds?.[0]},
+      hash: location.hash,
+    });
+  }, [location.hash, location.query, location.pathname, projectIds, router]);
+
+  useEffect(() => {
+    setProjId();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
This might be a good place for `browserHistory.getCurrentLocation()`:
i'm tinkering: 

```suggestion
import {browserHistory} from 'react-router';
...
  useEffect(() => {
    const location = browserHistory.getCurrentLocation();
    if (!location.query.project) {
      browserHistory.push({
        pathname: location.pathname,
        query: {...location.query, project: projectIds?.[0]},
        hash: location.hash,
      });
    } 
  }, [projectIds]);
```

Reply:
gave it a try and the odd thing is that even though the URL is setting the project, it's still rendering the wrong project - super weird

Reply:
:( kk cool

Reply:
maybe the suggestion isn't triggering the correct 'location updated' callback like i assumed